pilot/docker/Dockerfile.proxyv2

```diff
@@ -52,5 +52,8 @@ COPY envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
 COPY envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 COPY envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
 
+COPY stats-filter.wasm /etc/istio/proxy/stats-filter.wasm
```

Comment:
We want this to be /etc/istio/proxy/extensions 
That way we / customer can mount over this dir.

Reply:
Yeah, looks like mounting on the same path will remove these files. Shall we reserve `/etc/istio/proxy/extensions` for user provided plugins and keep these two plugins where they are now?

Reply:
Overmounting indeed replaces the directory , it is not unionfs.

Rethinking, We should treat the two wasm modules  as if they are compiled in. Given that, it does not have to be in the extensions dir, since we expect extension names to be versioned and these names are not.

You can keep it as it is, I will change it later if needed .